app/src/main/java/xyz/whereuat/whereuat/AtResponseInitiateReceiver.java

```diff
@@ -37,6 +37,7 @@ public void onReceive(final Context context, final Intent intent) {
         AsyncExecutor.service.submit(new Runnable() {
             @Override
             public void run() {
+                LocationProviderService.updateLocation(null);
```

Comment:
why not just make this implicit in the `getLocation` function. Reduce coupling and all that SDD stuff